src/frequently-asked-questions-about-fsrs.md

```diff
@@ -0,0 +1,45 @@
+# Frequently asked questions about FSRS
+
+***
+
+Q1: Which platforms support FSRS?
+
+A1: FSRS is supported on all platforms: desktop Anki (Windows, Mac, and Linux), AnkiWeb (browser), AnkiMobile (iOS), and AnkiDroid (Android). Make sure that your Anki(Mobile) version is 23.10 or newer, and your AnkiDroid version is 2.17 or newer.
+
+***
+
+Q2: My first interval for **Good** is 3-5 days, and my first interval for **Easy** is over a week long! Is this normal?
+
+A2: Yes. FSRS tends to give longer first intervals than SM-2, but for mature cards the opposite is true - FSRS is more conservative.
+
+If you wish to make the intervals shorter, you can increase your desired retention. But note that this will shorten all the intervals, not just the first intervals.
+
+***
+
+Q3: I only use **Again** and **Good**, will FSRS work fine?
+
+A3: Yes. In some cases, FSRS may even be more accurate if you only use 2 buttons.
+
+***
+
+Q4: Is it better to use the same parameters for all my decks or make several different presets with different parameters?
+
+A4: The answer to this question depends entirely on how similar your material is. For example, if you are learning Japanese and geography, it is recommended to use two different presets with different parameters. If you have two decks with Japanese vocabulary, you should use the same preset for both of them. Generally speaking, it's usually better to have different presets.
+
+***
+
+Q5: How often should I re-optimize parameters?
+
+A5: Once per month should be more than enough. A more sophisticated rule is to optimize every time the number of reviews doubles: after you did 100 reviews, then after 200, then after 400, etc. But the "one month" rule is simpler.
+
+***
+
+Q6: Does FSRS take into account the amount of time that I spend reviewing a card?
+
+A6: No, FSRS only uses interval lengths and grades. However, the amount of time you spend on reviews is used when calculating optimal retention using the "Compute minimum recommended retention" feature.
+
+***
+
+Q7: Do I need 1000 reviews before I can optimize parameters?
+
+A7: That was the case in earlier versions of Anki. In the most recent version, the optimizer can be used with any number of reviews.
```

Comment:
The threshold is removed in Anki 24.06.3.

Reply:
Do you want me to specify the exact version 
?

Reply:
Do it. There are a lot of people on previous versions. Apparently cuz of add ons.